Project-3/Project#3_tf

```diff
@@ -0,0 +1,292 @@
+
+6. Using Protege, place these in the BFO hierarchy where you think they fit best:
+
+  (a) Bach's Well-Tempered Clavier
+  
+Generically dependent continuant (e.g.  computer files), object (e.g. music sheet), or process (occurrent, e.g. playing the song)
+
+  
+  (b) Chair of the UB Philosophy Department
+
+Role
+
+  (c) SARS-CoV-2
+
+Object/Material entity (e.g. virus) 
+
+n.b., for symptoms to exist (i.e., disposition) another body would be needed to be infected and display symptoms.
+
+  (d) Mexico City
+
+Site or 3D spatial region or Instance
+
+  (e) The trunk of a minivan
+
+Site or fiat object part
+
+  (f) Occupation
+
+Role
+
+  (g) Ocean
+
+Site or 3D spatial region
+
+  (h) Lake
+
+Site or 3D spatial region
+
+
+7. True or False; explain your answers:
+
+  (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  
+True. “Immaterial entities listed under 1 (boundaries and sites) are in some cases continuant parts of their material.” (Smith 108)
+  
+  (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  
+False. You can have, as seen in (a), an instance of a ME that has an instance of an IE as part, but not the other way around. Concepts explain things, but things don't explain concepts--I think something along these lines grounds the direction of the relationship.
+  
+  (c) An organization may have another organization as part.
+  
+True. For example, the Federal Goverment has State governments as parts; or NATO has other Nation-States as a parts. (n.b., The relevancy or fit of the examples hinges, to some extent, on how on defines organization.) 
+  
+  (d) An organization may have no members as part. 
+  
+True and False. Again, this answer hinges on how on defines 'organization.' The 'perspectivalism' we read about a few weeks back seems to be useful here. So, for example, take the debate team of a college. The debate team is an organization. If (a) one defines organization relative to the entity being recognized by another entity/organization for its role, the debate club will exist even if it has no members as part provided that it continues to be recognized by the college according to its role. If, however, (b) one defines organization relative to the performance of some activity then it might make sense to view the debate team as an object aggregate, then without members as part the debate team there would be no activity performed, for it would have no performers, and therefore the object aggregate would not exist. So, according to (a) the debate team, grounded in institutional recognition of its role, may exist even while having no members as part. The answer according to (a) is, therefore, true. According to (b), however, the answer is false. If the debate club is viewed as an object aggregate because as object aggregate is the sum of its parts, it loses its necessary parts to being an object aggregate by virtue of having no members as pars.
```

Comment:
Strictly, we allow for organizations to have no members assuming they at some point had members. Nevertheless, excellent conceptual analysis.